local-process-quickstart.sh

```diff
@@ -48,18 +60,18 @@ cleanupFunction()
    echo ""
    echo "Setting the Kube context to $AKSNAME in $RGNAME"
    az aks get-credentials -g $RGNAME -n $AKSNAME
-   $HELMDIR/helm --namespace dev uninstall bikesharingapp
-   $HELMDIR/helm --namespace dev uninstall $INGRESSNAME
-   echo "Delete namespace dev? (Y/n) : "
+   $HELMDIR/helm --namespace $BIKENS uninstall bikesharingapp
+   $HELMDIR/helm --namespace $BIKENS uninstall $INGRESSNAME
+   echo "Delete namespace $BIKENS? (Y/n) : "
    read RESPONSE
    if [ "$RESPONSE" == "y" ] || [ "$RESPONSE" == "Y" ]; then
-      kubectl delete namespace dev
+      kubectl delete namespace $BIKENS
    fi
    rm -rf $HELMDIR
    exit 0
 }
 
-while getopts "g:n:r:cd" opt; do
+while getopts "g:n:rcd" opt; do
```

Comment:
Was this an intentional change? If the `r` option needs a value, it needs the `:`

Reply:
r defaults to PWD, so it didn't feel necessary to have it there.

Reply:
Isn't the intended usage like `-r ../some/directory/that/isnt/pwd`? The getopts string needs to be `r:` for that to work. Or else it should be removed as an option.